test/onnx/onnx_test.cpp

```diff
@@ -2665,6 +2667,34 @@ TEST_CASE(if_else_test)
     EXPECT(p == prog);
 }
 
+TEST_CASE(if_else_test_inlined)
+{
+    migraphx::program p;
+    auto* mm = p.get_main_module();
+    migraphx::shape sc{migraphx::shape::bool_type, {1}};
+    mm->add_literal(migraphx::literal(sc, {0}));
+    migraphx::shape s{migraphx::shape::float_type, {2, 3}};
+    std::vector<float> ones(s.elements(), 1.0f);
+    mm->add_literal(s, ones);
+    std::vector<float> rand = {0.811412, -0.949771, -0.169276, 0.36552, -0.14801, 2.07061};
```

Comment:
I like how program is structured in `if_else_test` or `if_literal_test`, separating literal values'  declaration and construction of MIGraphX program. If you can re-arrange it would be good.

Reply:
Sure I'll clean this up

Reply:
Done